pages/index.tsx

```diff
@@ -2,19 +2,19 @@ import BetterSocialIcon from 'component/Brand/BetterSocialIcon';
 import Constant from 'utils/constant';
 import React from 'react'
 import UserAgentUtils from 'utils/userAgent'
+import getConfig from 'next/config';
 import parser from 'ua-parser-js';
 import { BaseContainer } from "component/Page/BaseContainer";
 import { Helmet } from 'react-helmet';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements are not organized. It's a good practice to group and sort import statements. First, import libraries/modules, then local files. Also, keep the relative imports together.

```diff
- import Constant from 'utils/constant';
- import React from 'react'
- import UserAgentUtils from 'utils/userAgent'
- import getConfig from 'next/config';
- import parser from 'ua-parser-js';
- import { BaseContainer } from "component/Page/BaseContainer";
- import { Helmet } from 'react-helmet';
+ import React from 'react'
+ import { Helmet } from 'react-helmet';
+ import parser from 'ua-parser-js';
+ import getConfig from 'next/config';
+ import Constant from 'utils/constant';
+ import UserAgentUtils from 'utils/userAgent'
+ import { BaseContainer } from "component/Page/BaseContainer";
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->